pkg/pagerduty/service.go

```diff
@@ -319,12 +319,18 @@ func (c *SvcClient) CreateService(data *Data) (string, error) {
 		if newSvc == nil {
 			return "", fmt.Errorf("creating service %v returned an empty response: %w", clusterService.Name, err)
 		}
+
+		err = nil
 	}
 	data.ServiceID = newSvc.ID
 
-	data.IntegrationID, err = c.createIntegration(newSvc.ID, "V4 Alertmanager", "events_api_v2_inbound_integration")
-	if err != nil {
-		return "", fmt.Errorf("unable to create integration for service %v: %w", newSvc.ID, err)
+	if len(newSvc.Integrations) == 0 {
+		data.IntegrationID, err = c.createIntegration(newSvc.ID, "V4 Alertmanager", "events_api_v2_inbound_integration")
+		if err != nil {
+			return "", fmt.Errorf("unable to create integration for service %v: %w", newSvc.ID, err)
+		}
+	} else {
+		data.IntegrationID = newSvc.Integrations[0].ID
```

Comment:
**Suggestion:** We should probably differentiate here between `len(newSvc.Integrations) == 1` and `len(newSvc.Integrations) > 1`. If there's more than 1, we probably want to log a warning.
If a service has multiple integrations, are we sure the first one is the correct one?

Reply:
We could check againt the integration type... But it is a string... and the string returned by the PagerDuty API call is NOT the one provided when creating the Integration object